public/static/partials/user_host_options.html

```diff
@@ -92,7 +92,7 @@
              Volume:
             </span>
             <strong>
-              [[ homeVolumeID ? homeVolumeID : "New Volume" ]]
+              [[homeVolumeDisplay]]
```

Comment:
I think the previous approach of having the display logic here works better. In the scenario that you miss updating the homeVolumeDisplay variable, the previous approach would still display the right thing